### PR TITLE
Support skipping control queries in verifier

### PR DESCRIPTION
**Pull request by arhimondr**
## Description

Useful when trying to use verifier as a simple tool for running a list of queries without any additional verifications

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Verifier

> How would you describe this change to a non-technical end user or system administrator?

`-`

## Related issues, pull requests, and links

`-`

## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: service/trino-verifier/src/main/java/io/trino/verifier/Validator.java [hunk 1]
```diff
@@ -539,13 +545,13 @@ private QueryResult executeQuery(String url, String username, String password, Q
...
-            return new QueryResult(state, exception, null, null, queryId, ImmutableList.of());
+            return new QueryResult(state, exception, nanosSince(start), queryCpuTime, queryId, ImmutableList.of());
         }
         catch (VerifierException e) {
-            return new QueryResult(State.TOO_MANY_ROWS, e, null, null, queryId, ImmutableList.of());
+            return new QueryResult(State.TOO_MANY_ROWS, e, nanosSince(start), queryCpuTime, queryId, ImmutableList.of());
         }
         catch (UncheckedTimeoutException e) {
-            return new QueryResult(State.TIMEOUT, e, null, null, queryId, ImmutableList.of());
+            return new QueryResult(State.TIMEOUT, e, nanosSince(start), queryCpuTime, queryId, ImmutableList.of());
```
Comment (losipiuk) [inline service/trino-verifier/src/main/java/io/trino/verifier/Validator.java:554]: those look unrelated - could be separate commit.

**Review by losipiuk — APPROVED** [review]

**Review by linzebing — APPROVED** [review]

File: service/trino-verifier/src/main/java/io/trino/verifier/Validator.java (hunk 1 above)
Comment (arhimondr) [reply service/trino-verifier/src/main/java/io/trino/verifier/Validator.java:554]: This is a part of the `Record query id for failed queries in verifier`. Maybe i should change the commit message to something like `Record query id and runtime information for failed queries in verifier`?
